launchable/commands/record/tests.py

```diff
@@ -80,19 +81,39 @@
 @click.pass_context
 def tests(context, base_path: str, session: Optional[str], build_name: Optional[str], post_chunk: int, subsetting_id: str,
           flavor, no_base_path_inference):
+    org, workspace = get_org_workspace()
+
+    if org is None or workspace is None:
+        raise click.UsageError(click.style(
+            "Could not identify Launchable organization/workspace. "
+            "Please confirm if you set LAUNCHABLE_TOKEN or LAUNCHABLE_ORGANIZATION and LAUNCHABLE_WORKSPACE environment variables",
+            fg="red"))
+
+    client = LaunchableClient(test_runner=context.invoked_subcommand,
+                              dry_run=context.obj.dry_run)
+    res = client.request("get", "verification")
```

Comment:
Why not call the verify method instead of sending the request directly to make exception handling and messages common? You can leave the checking of org  and work to them too.

Reply:
I don't think calling the command line method from another command line method is well-organized way since each command line has its own logic, ends up in change in `verify` command line affecting `record tests` unintentionally. Rather, I propose to make an independent `verify` method outside of command lines to be used generally to ensure the auth and connection are all fine beforehand.